frontend-react/src/resources/SubmissionsResource.ts

```diff
@@ -6,10 +6,12 @@ const { RS_API_URL } = config;
 
 type SubmissionsResourceParams = {
     organization: string;
-    pageSize: number;
+    sortdir: string;
```

Comment:
Completelyyyyyyyy unrelated to this changeset, but I feel like the query strings would be easier to remember/less likely to be mistaken if we had consistent camel-casing.  It looks like there are a few query parameters (in [here](https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/src/main/kotlin/history/azure/ReportFileFunction.kt) at least?) that could be made consistent.  Again, not related to this changeset -- just a thought that occurred randomly in my mind! :P

Reply:
Ha, I actually brought this up today with another team-member for a different API. I agree, I think query params would be good here but out of scope for this change. Platform team will keep this in mind going forward.

Reply:
Glad to hear it, and I'm looking forward to the great changes the Platform team will be making 🙌 🙌 🙌